tests/forms_tests/tests/test_media.py

```diff
@@ -526,48 +535,53 @@ def test_html_safe(self):
 
     def test_merge(self):
         test_values = (
-            (([1, 2], [3, 4]), [1, 2, 3, 4]),
+            (([1, 2], [3, 4]), [1, 3, 2, 4]),
```

Comment:
The whole test should be updated to a list of list and a result list.
eg:
```python
([[1,2],[1,3],[2,3], [5,7], [5,6], [6,7,9], [8,9]], [1,2,3,5,6,7,8,9])
```

Reply:
Good idea. I added such a test [here](https://github.com/django/django/pull/11019/commits/2fcd267ecce6609af4e5910742f5b61f313263ab)

(Aside: The resulting list is `[1, 5, 8, 2, 6, 3, 7, 9]`)